.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: generic
+services:
+  - docker
+env:
+  - PYTHON=python3.5
+  - PYTHON=python3.6
+
+install:
+  - docker build -t journalwatch/ci ci/"$PYTHON"
+script:
+  - docker run -v $(pwd):/build/ journalwatch/ci /usr/bin/"$PYTHON" /build/test_journalwatch.py
```

Comment:
Why `$(pwd)` and not `$PWD`?

Reply:
Also, you're running the Python file here, which does nothing. You'll need to run `"$PYTHON" -m pytest /build/test_journalwatch.py` or so instead, and also install that in the Dockerfile (e.g. via `pip install pytest`)

Reply:
> Why $(pwd) and not $PWD?

I did not know about `$PWD`. ;)

> you're running the Python file here, which does nothing.

Oh. Maybe the README should say that. This is also how I ran the tests before submitting the other PRs.^^

> I'd prefer to get the newest version via pip though.

Sure. Same for python3-systemd?